content/en/blog/_posts/2021-11-23-object-storage-support.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: 'Introducing COSI: Object Storage Support in Kubernetes'
+date: 2021-11-23
```

Comment:
```suggestion
date: 2021-12-14
```

Reply:
Per the request of the K8s 1.23 Release Comms Team, please update the publishing date to 12/14. Thank you!